dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -667,17 +712,18 @@ def test_correct_total_sample_size_and_counts_and_mutability(self):
             # Profile Twice
             profile.update_profile(data)
 
+            # rows sampled are [5, 6] (0 index)
             self.assertEqual(16, profile.total_samples)
             self.assertEqual(4, profile._max_col_samples_used)
-            self.assertEqual(2, profile.row_has_null_count)
-            self.assertEqual(0.5, profile._get_row_has_null_ratio())
-            self.assertEqual(2, profile.row_is_null_count)
-            self.assertEqual(0.5, profile._get_row_is_null_ratio())
+            self.assertEqual(1, profile.row_has_null_count)
+            self.assertEqual(0.25, profile._get_row_has_null_ratio())
+            self.assertEqual(1, profile.row_is_null_count)
+            self.assertEqual(0.25, profile._get_row_is_null_ratio())
             self.assertEqual(0.4375, profile._get_unique_row_ratio())
```

Comment:
since update repeats the same rows, should have been caught as 1 not 2 null rows